prep-css-box-model/style.css

```diff
@@ -0,0 +1,13 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border-style: double, darkgrey, 3px;
```

Comment:
Hi Joel, try removing the commas and changing your property to `border` to get the border styles to apply.